src/psyclone/tests/psyGen_test.py

```diff
@@ -50,8 +50,9 @@
 import pytest
 from fparser import api as fpapi
 from psyclone.core.access_type import AccessType
+from psyclone.psyGen import GlobalSum, InvokeSchedule
```

Comment:
We now have two separate `from psyclone.psyGen import ...` lines.

Reply:
pylint also complains that `InvokeSchedule` is imported in other places in this file too.

Reply:
Updated.